lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -73,7 +73,12 @@
         <div class="start-certificate-generation">
             <h2>${_("Generate Certificates")}</h2>
             <form id="certificates-generating-form" method="post" action="${section_data['urls']['start_certificate_generation']}">
-                <input type="button" id="btn-start-generating-certificates" value="${_('Generate Certificates')}" data-endpoint="${section_data['urls']['start_certificate_generation']}"/>
+                % if section_data['html_cert_enabled'] and section_data['active_certificate'] is None:
+                    <p>${_("Course certificate generation requires an activated web certificate configuration.")}</p>
+                    <button id="disabled-btn-start-generating-certificates" class="is-disabled" disabled>${_('Generate Certificates')}</button>
```

Comment:
@ziafazal I used the button because i think 'is-disable' css is comes under button hierarchy. i saw disabled button somewhere in the same file.